backend/tests/test_security_monitor_config.py

```
@@ -0,0 +1,302 @@
+"""
+Tests for configurable SecurityMonitor thresholds and allowlist functionality.
+"""
+
+import os
+import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)